docs/integrations/yaml-file.md

```diff
@@ -87,6 +87,10 @@ An explanation of these fields is as follows:
 | **required** | Boolean value to indicate that the parameter is required |
 | **additionalinfo** | Additional info about the field, will appear under a question mark in the configuration pannel |
 
+Integration parameters may be hidden from the XSOAR UI, using the optional `hidden` field.
+- To hide the parameter in all marketplaces (XSOAR, XSIAM), use a boolean `true`. 
+- To hide the parameter in specific content marketplace versions, provide list of marketplace version names (e.g. `xsoar` or `marketplacev2` for XSIAM)
```

Comment:
```suggestion
- To hide the parameter in specific content marketplace versions, provide list of marketplace version names (e.g. `[xsoar]` or `[marketplacev2]` for XSIAM)
```

Reply:
the syntax is 
```
hidden:
  -  xsoar
  - marketplacev2
```

which I didn't know how to describe in a one-liner (or whether it's worth a code-block for the example), wdyt?

Reply:
Let's leave it like this.